### PR TITLE
added SwissHQ

### PR DESCRIPTION
**Pull request by klirman**
### General:
* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated metadata.json`)*
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [x] Folder with appropriate name has been created / updated (must not include any non-alpha characters or whitespace).
* [x] No changes were made to the file's formatting (4 spaces per tab).
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping matching my id (check if there is an existing folder).
* [x] My field values match your requirements.
* You can view our patterns here: [metadata.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/metadata.schema.json), or take a look below and complete the field checklist:
  - [x] `id`: a lowercase string, which should match the folder name *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. `["my.server", "your-server.com"]`)*
    - You do not need to specify subdomains, Lunar Client services automatically detect them.
  - [x] `primaryAddress`: an address *(ex. `my.server`)*
  - [x] `minecraftVersions`: an array with Minecraft versions as strings *(ex. `["1.18", "1.19"]` - Must be versions supported by Lunar Client)*
  - [x] `primaryMinecraftVersion`: a Minecraft version as a string *(ex. `1.18` - Must be versions supported by Lunar Client)*
  - [x] `primaryColor`: a hexademical color code that primarily distiguishes the server *(ex. `#00FFFF`)* 
  - [x] `secondaryColor`: a hexademical color code that accompanies the `primaryColor` of the server *(ex. `#FF0000`)*
  - [x] `primaryRegion`: the primary region where your server operates in *(ex. `NA`)*
  - [x] `regions`: a list of regions where you have servers located that service your players *(ex. `["NA", "EU", "AS"]`)*
  - [x] `gameTypes`: a list of games that describe the content on your server, must be a max of 3 listed *(ez. `["PVP", "UHC", "HCF"]`)*

#### Logo
* [x] My logo is a `png` file.
* [x] I have uploaded my logo to my server folder *(ex. `lunarnetwork`)* and named it `logo.png`.
* [x] My logo has a transparent background and is square (1:1 aspect ratio).
* [x] My logo is `512` pixels in width and height.
* [x] My logo is my own property and complies with relevant copyright/privacy laws.

#### Background
* [x] My background is a `png` file.
* [x] I have uploaded my background to my server folder *(ex. `lunarnetwork`)* and named it `background.png`.
* [x] My background has an aspect ratio of 16:9.
* [x] My background is atleast `1920` pixels in width and `1080` pixels in height (can be bigger, but must remain the same aspect ratio).
* [x] My background contains relevant content pertaining to the server, is my own property, and complies with relevant copyright/privacy laws.
* [x] My background doesn't contain any markings, text, or logos (unless part of a build).

### CODE REVIEW

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please make the following changes:
- Upload a background that represents your server
- Upload a logo that is actually for your server (this is MineHQs logo)
- Fix / validate your json formatting and add all missing fields to your metadata file

**Comment by klirman** [comment]
All done.
> * Upload a background that represents your server
this represents the server perfectly fine; it's a picture from Zurich, one of the major cities of switzerland.

**Review by itsJordan — CHANGES_REQUESTED** [review]
Could you please upload a logo that actually represents your server and that you have the rights to use?

**Comment by klirman** [comment]
It  represents the server just fine. the entire theme of this server is it being Swiss, and what better to use as a logo than the actual flag of Switzerland? nothing would represent it better.

And yes, I do have the rights to use this. 
image

[Source](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3663670/#:~:text=From%20an%20international%20copyright%20law,copyright%20%5B8%2C9%5D.)

**Review by itsJordan — CHANGES_REQUESTED** [review]
Could you please upload a version of your background that is 1920x1080

File: servers/swisshq/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "id": "swisshq",
+    "name": "SwissHQ Network",
+    "addresses": [
+        "swisshq.net",
```
Comment (itsJordan) [inline servers/swisshq/metadata.json:5]:
```suggestion
        "swisshq.net"

```

File: servers/swisshq/metadata.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    ],
+    "primaryAddress": "swisshq.net",
+    "minecraftVersions": [
+        "1.7",
+        "1.8"
+    ],
+    "primaryMinecraftVersion": "1.7",
+    "primaryColor": "#ffffff",
+    "secondaryColor": "##ff0000",
```
Comment (itsJordan) [inline servers/swisshq/metadata.json:14]:
```suggestion
    "secondaryColor": "#ff0000",

```

File: servers/swisshq/metadata.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "primaryLanguage": "en",
+    "languages": [
+        "en"
+    ],
+    "primaryRegion": "EU",
+    "regions": [
+        "EU",
```
Comment (itsJordan) [inline servers/swisshq/metadata.json:21]:
```suggestion
        "EU"

```

File: servers/swisshq/metadata.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+    "id": "swisshq",
+    "name": "SwissHQ Network",
+    "addresses": [
+        "swisshq.net"
+
```
Comment (itsJordan) [inline servers/swisshq/metadata.json:6]:
```suggestion

```

**Comment by itsJordan** [comment]
This will be ready to merge once a valid background is supplied.

**Review by itsJordan — APPROVED** [review]

**Pull request merged**